packages/vitest/src/node/config/resolveConfig.ts

```diff
@@ -213,9 +213,8 @@ export function resolveConfig(
       && resolved.poolOptions?.threads?.singleThread
     const isSingleFork
       = resolved.pool === 'forks' && resolved.poolOptions?.forks?.singleFork
-    const isBrowser = resolved.browser.enabled
 
-    if (resolved.fileParallelism && !isSingleThread && !isSingleFork && !isBrowser) {
+    if (resolved.fileParallelism && !isSingleThread && !isSingleFork) {
```

Comment:
The requirement for `--no-file-parallelism` when used with `--browser` was removed in https://github.com/vitest-dev/vitest/pull/6433 but during this PR I noticed that the flag actually affects browser mode as well. We need it in order to successfully set breakpoint to the first test file.